package.json

```diff
@@ -29,16 +30,19 @@
     "tslib": "^2.2.0"
   },
   "devDependencies": {
+    "@peculiar/webcrypto": "^1.1.7",
```

Comment:
Would you like to try this https://nodejs.org/api/webcrypto.html?

Reply:
Did you try it?

Reply:
NO, the `SubtleCrypto` is added in NODEJS **15.0.0**

Reply:
So what's wrong? Can't you just use Node 15 or 16?